.eslintrc

```diff
@@ -18,6 +18,7 @@
   "rules": {
     "func-names": 0,
     "no-underscore-dangle": 0,
+    "no-confusing-arrow": 0,
```

Comment:
Maybe would be better to disable this only for the line I did not want to be considered as an error, but it was not working...  Anyway, this is not the most dangerous error I can have and normally I do not overuse arrow functions, so I think this change would be fine... 🙈